source/acquaman/REIXS/REIXSXESScanController.cpp

```diff
@@ -182,6 +182,16 @@ bool REIXSXESScanController::startImplementation() {
 	return true;
 }
 
+void REIXSXESScanController::pauseImplementation()
+{
+	REIXSBeamline::bl()->mcpDetector()->pauseDwelling();
+}
+
+void REIXSXESScanController::resumeImplementation()
+{
+	REIXSBeamline::bl()->mcpDetector()->resumeDwelling();
+}
```

Comment:
More importantly, there are no `setPaused()` or `setResumed()` calls.  @XuetaoLiu-CLS, did you manage to get it into these states in testing?